objc/core/A0LockReact.m

```diff
@@ -44,6 +49,19 @@ - (void)configureLockWithClientId:(NSString *)clientId domain:(NSString *)domain
     _lock = [A0Lock newLockWithClientId:clientId domain:domain];
 }
 
+- (void)hideWithCallback:(A0LockCallback)callback {
+    if (self.shown) {
```

Comment:
I am not convinced we should add this flag, the worst case scenario is that you'll dismiss a VC that is presented from the rootViewController of the window if Lock is not displayed (which should be strange). So I'd rather remove this flag and just dismiss the presented VC.